cbpi/extension/mqtt_actor/__init__.py

```diff
@@ -1,45 +1,90 @@
 # -*- coding: utf-8 -*-
+
+# Topic pattern:
+# Tasmota on sonoff:
+# tasmota on sonoff TH16 or TH10 topic: cmnd/<tasmota_name_of _device>/Power
+# or like eg. cmnd/tasmota_F9713B/Power
+# Power in the meaning of on/off. Power as % is not supported
+# Have a look at https://stevessmarthomeguide.com/setting-up-the-sonoff-tasmota-mqtt-switch/
+# In Tasmota adjust configuration-> configure logging->Telemetry period (300) from 300s to 10s
+# to speed up the sensor (I know this is the actor section)
+# If you need to change payload please edit in function async def on(..) and async def off..)
+#
+# EspEasy
+# use this link:
+# https://nerdiy.de/howto-espeasy-mqtt-server-konfigurieren-und-topics-abonnieren/
+# You may use translate function of the browser to translate the language
+# If you need to change payload please edit in function async def on(..) and async def off..)
+
 import asyncio
+import logging
 import json
-from cbpi.api import parameters, Property, CBPiActor
 from cbpi.api import *
 
-@parameters([Property.Text(label="Topic", configurable=True, description = "MQTT Topic")])
+logger = logging.getLogger(__name__)
+
+
+@parameters([Property.Text(label="Topic", configurable=True, description="MQTT Topic"),
+             Property.Select(label="select_Mqtt_System", options=["default", "tasmota on sonoff THx", "ESPEasy"],
```

Comment:
Just an idea:
Wouldn't it be more flexible to provide templates (as a string) for the payload? This way this implementation would not be tied to specific devices an could be provided in the settings.

e.g.
```python
@parameters([ ...
    Property.Text(label="Payload", configurable=True, description="Defines the payload that is sent to the actor. Uses placeholders for actual values."])
```

Then you could replace the placeholders before sending the payload.

Reply:
Yes, I thought that too. For me as a python beginner, this became too difficult, as a payload for on and a payload for off had to be generated. Each optionally also as a json. Combined with power or not power, I recently fought with dicts and quotation marks. In the end, I made it easy for myself. I agree with you that it is not universal. So far the community has got by with it. Most have adapted the add-in, which is now easier. It would be better if we could incorporate most of the combinations through feedback from the community.

Reply:
I made a draft PR for an alternative that implements my previous suggestion. #14.

@avollkopf Just wanted to suggest this. You can still use this implementation if this suits better, of course. No hard feelings here.